### DISCUSSION TITLE
Welcome to Prime Discussions!

### DISCUSSION THREAD

**Discussion by guzhenhuaGitHub** [guzhenhuaGitHub/Prime#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kekko82** [comment]
hello everyone, I have a problem, after replacing the camera of the dji mini 2, it doesn't work and I would like to know the code to type with python to pair a new camera with the drone in order to make it work. Please help me
